### PR TITLE
Update support.md

### PR DESCRIPTION
**Pull request by marcogrcr**
Updated permissions using the following reference: https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssupport.html

**By opening a Pull Request, you agree that your contribution is licensed under CC0 1.0 Universell (CC0 1.0)**

Please make sure to provide all the fields! Example below!

* Service: `support`
* Action: `AddAttachmentsToSet`, `AddCommunicationToCase`, `CreateCase`, `DescribeAttachment`, `DescribeCaseAttributes`, `DescribeCases`, `DescribeCommunications`, `DescribeIssueTypes`, `DescribeServices`, `DescribeSeverityLevels`, `DescribeSupportLevel`, `InitiateCallForCase`, `InitiateChatForCase`, `PutCaseAttributes`, `RateCaseCommunication`, `ResolveCase`, and `SearchForCases`.
* Action documentation:
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddAttachmentsToSet.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddCommunicationToCase.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_CreateCase.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeAttachment.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeCases.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeCommunications.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeServices.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeSeverityLevels.html
  * https://docs.aws.amazon.com/awssupport/latest/APIReference/API_ResolveCase.html
* Resources: `*`
* Conditions:  _(none)_
* Source: https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssupport.html

### CODE REVIEW

File: services/support.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
 | --- | --- | --- | --- |
-| [support:*](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html#accessing-support) | Support does not let you allow or deny access to individual actions; therefore, the Action element of a policy is always set to support:*. Similarly, Support does not provide resource-level access, so the Resource element is always set to *. An IAM user with Support permissions has access to all Support operations and resources. | * | - |
-
+| [support:AddAttachmentsToSet](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddAttachmentsToSet.html) | Adds one or more attachments to an AWS Support case. | * | - |
+| [support:AddCommunicationToCase](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_AddCommunicationToCase.html) | Adds a customer communication to an AWS Support case. | * | - |
+| [support:CreateCase](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_CreateCase.html) | Creates a new AWS Support case. | * | - |
+| [support:DescribeAttachment](https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeAttachment.html) | Returns the description for an attachment. | - |
```
Comment (michaelwittig) [inline services/support.md]: Should end with `| * | - |`?

**Review by michaelwittig — CHANGES_REQUESTED** [review]

**Comment by michaelwittig** [comment]
Thanks @marcogrcr ! I left one comment above.

**Comment by michaelwittig** [comment]
Thanks @marcogrcr

File: services/support.md (hunk 1 above)
Comment (marcogrcr) [reply services/support.md]: Yup, missed that by mistake. Added a new commit with that fixed.

**Pull request merged**